src/platforms/dotnet/guides/extensions-logging/index.mdx

```diff
@@ -92,7 +92,7 @@ Example using `appsettings.json`:
   "Sentry": {
     "Dsn": "___PUBLIC_DSN___",
     "MinimumBreadcrumbLevel": "Warning",
-    "MinimumEventLevel": "Critical",
+    "MinimumEventLevel": "Error",
```

Comment:
What about we lower the values then to serve as an example:

Set `MinimumBreadcrumbLevel` to `Debug` and:

```suggestion
    "MinimumEventLevel": "Warning",
```